lib/rspec/matchers/fail_matchers.rb

```diff
@@ -14,8 +14,8 @@ module FailMatchers
       #
       # @example
       #   expect { some_expectation }.to fail
-      def fail
-        raise_error(RSpec::Expectations::ExpectationNotMetError)
+      def fail(&block)
+        raise_error(RSpec::Expectations::ExpectationNotMetError, &block)
```

Comment:
No.  Why would it be?  "fail" in rspec-expectations refers to a `RSpec::Expectations::ExpectationNotMetError` whereas in rspec-mocks it refers to a `RSpec::Mocks::MockExpectationError` so there's no common meaning of "fail" across libs.